app/src/main/java/endava/codebase/android/movieapp/ui/component/FavoriteButton.kt

```diff
@@ -16,17 +18,20 @@ import endava.codebase.android.movieapp.ui.theme.*
 
 @Composable
 fun FavoriteButton(
-    modifier: Modifier = Modifier,
+    modifier: Modifier = Modifier
+        .size(30.dp)
+        .padding(MaterialTheme.spacing.small),
```

Comment:
Default modifier should be empty, size and padding should be defined from outside